crab_staticanalysis/build_run.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# when building the image manually, push it to registry.cern.ch
+IMAGETAG=registry.cern.ch/cmscrab/crabstaticanalysis:latest
+# gitlab will automatically build this image and keep it in its registry.
+# IMAGETAG=gitlab-registry.cern.ch/cmsdocks/dmwm:crab_staticanalysis
+
+docker build --no-cache -t $IMAGETAG .
+
+# docker run -it $IMAGETAG 
+
+# test CRABServer
+mkdir -p ~/temp/artifacts-0 ~/temp/artifacts-1
+docker run \
+-e ghprbPullId=7309 -e  ghprbTargetBranch=master \
```

Comment:
Do you want to make this pullId parameterized (from the jenkins job)?

Reply:
I use the script `./build_run.sh` to build and run the tests on my laptop, mainly for debugging purposes, and I chose to provide reasonable defaults.

We would use the image that I am adding with this PR in the jenkins job https://cmssdt.cern.ch/dmwm-jenkins/view/CRAB/job/DMWM-CRABServer-PR-pylintpy3/, which takes automatically those env variables (`ghprbPullId`, `ghprbTargetBranch`, ...) form the upstream jobs [1], one for each repository:

- CRABClient: https://cmssdt.cern.ch/dmwm-jenkins/view/CRAB/job/DMWM-CRABClient-PR-test/
- CRABServer: https://cmssdt.cern.ch/dmwm-jenkins/view/CRAB/job/DMWM-CRABServer-PR-test/

I already tested the whole jenkins pipeline and it is going ok. I built the image myself and pushed it to registry.cern.ch. After we merge this PR, i will use the image built automatically from gitlab-registry.cern.ch

[1] thanks to the [jenkins' github plugin](https://plugins.jenkins.io/ghprb/#plugin-content-environment-variables)